cli/command/image/tree.go

```diff
@@ -281,11 +282,21 @@ func printNames(out *streams.Out, headers []imgColumn, img topImage, color, unta
 		_, _ = fmt.Fprint(out, headers[0].Print(untaggedColor, "<untagged>"))
 	}
 
-	for nameIdx, name := range img.Names {
-		if nameIdx != 0 {
-			_, _ = fmt.Fprintln(out, "")
+	namesLongestToShortest := slices.SortedFunc(slices.Values(img.Names),
```

Comment:
I think we should probably upgrade the required Go version in `vendor.mod` to 1.23 since this function was only added then.
https://pkg.go.dev/slices#SortedFunc

Reply:
Ah good catch!

We tend to target the oldest supported Go release, so we won't move to 1.23 until 1.24 is released.
I'll replace it with a different implementation.